### PR TITLE
Better document how to support IE8; Document browser support

### PR DESCRIPTION
**Pull request by 36degrees**
I suggest stepping through these commits one by one – 595ff06 wraps the README consistently but this makes the overall diff hard to review.

### CODE REVIEW

**Comment by igloosi** [comment]
says outdated but url is still broken here https://github.com/alphagov/govuk-frontend/blob/d40d824aec91bc2e80eae291c3a0b489928652d0/README.md#browser-support

File: README.md [hunk 1]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
 
 Copy and paste code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
 
+## Browser support
+
+As a minimum, GOV.UK Frontend will allow you to build services that
+comply with the [guidance in the service manual](https://www.gov.uk/service-
+manual/technology designing-for-different-browsers-and-devices#browsers-to-test-
```
[the remark this answers is not in the document]
Comment (36degrees) [reply README.md]: Updated – https://github.com/alphagov/govuk-frontend/blob/f48618982150b7ab31d826727042200d11990547/README.md#browser-support

**Comment by 36degrees** [comment]
I've asked @amyhupe to review this as quite a lot has changed.

File: docs/supporting-internet-explorer-8.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# Supporting Internet Explorer 8
+
+If you are including GOV.UK Frontend as part of your application's stylesheets
+then you will need to do some additional work to support Internet Explorer 8.
```
Comment (amyhupe) [inline docs/supporting-internet-explorer-8.md]: Can you put (IE8) in brackets after Internet Explorer 8? That will make it clearer when we refer to it as IE8 hereafter.

File: docs/supporting-internet-explorer-8.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+1. [Generate an IE8 specific stylesheet](#bundling-an-ie8-specific-stylesheet)
+2. [Transform the IE8 stylesheet using oldie](#transforming-the-ie8-stylesheet-using-oldie)
+3. [Include the IE8 stylesheet in your project](#including-the-ie8-stylesheet-in-your-project)
+
+You will then be able to [write CSS that targets IE8 in your own application
```
Comment (amyhupe) [inline docs/supporting-internet-explorer-8.md]: Change: 

You will then be able to [write CSS that targets IE8 in your own application...

to

Once you have completed these steps, you will be able to [write CSS that targets IE8 in your own application...

File: docs/supporting-internet-explorer-8.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+styles](#writing-styles-that-target-ie8).
+
+If you are using the distributed versions of GOV.UK Frontend that already
+include an IE8 specific stylesheet then you can include that in your project.
```
Comment (amyhupe) [inline docs/supporting-internet-explorer-8.md]: Change:

include an IE8 specific stylesheet

to:

include an IE8-specific stylesheet

File: docs/supporting-internet-explorer-8.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  min-width: 100px;
+
+  // Specify width for IE8 only
+  @include govuk-is-ie8 {
+    width: 100px;
+  }
+}
+```
+
+The `govuk-not-ie8` mixin can be used to wrap rules that you want to be excluded
```
Comment (amyhupe) [inline docs/supporting-internet-explorer-8.md]: Change: 

"rules that you want to be excluded"

to:

"rules that you want to exclude"

File: docs/supporting-internet-explorer-8.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+when generating the IE8 specific stylesheet.
```
Comment (amyhupe) [inline docs/supporting-internet-explorer-8.md]: Change to:

IE8-specific stylesheet.

File: README.md [hunk 7]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
 
 Copy and paste code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
 
+## Browser support
+
+As a minimum, GOV.UK Frontend will allow you to build services that
+comply with the [guidance in the service manual](https://www.gov.uk/service-
```
Comment (amyhupe) [inline README.md]: Write service manual in title case, so: Service Manual.

File: README.md [hunk 8]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
 
 Copy and paste code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
 
+## Browser support
+
+As a minimum, GOV.UK Frontend will allow you to build services that
```
Comment (amyhupe) [inline README.md]: Remove 'As a minimum'. Start sentence 'GOV.UK Frontend will allow...'

File: README.md [hunk 9]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
...
+comply with the [guidance in the service manual](https://www.gov.uk/service-
+manual/technology designing-for-different-browsers-and-devices#browsers-to-test-
+in).
+
+At the minute, GOV.UK Frontend officially supports the following browsers:
```
Comment (amyhupe) [inline README.md]: Change: 

'At the minute,...'

to:

'Currently,...'

File: README.md [hunk 10]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
...
+| macOS            | Google Chrome (latest 2 versions)      | compliant   |
+| macOS            | Mozilla Firefox (latest 2 versions)    | compliant   |
+| iOS              | Safari for iOS 9.3+                    | compliant   |
+| iOS              | Google Chrome (latest 2 versions)      | compliant   |
+| Android          | Google Chrome (latest 2 versions)      | compliant   |
+| Android          | Samsung Internet (latest 2 versions)   | compliant   |
+| Windows Phone    | Internet Explorer (latest 2 versions)  | compliant   |
+
+‘Compliant’ means that the components must look as good and function as well as
+it does in other modern browsers.
```
Comment (amyhupe) [inline README.md]: Change:

'it does'

to:

'they do'

File: README.md [hunk 11]
```diff
@@ -179,6 +179,42 @@ Add the CSS and JavaScript code to your HTML template:
...
+
+'Functional' means the components may not look perfect in the given browser, but
+must still be usable without errors and without 'looking broken'.
+
+If you are including GOV.UK Frontend as part of a stylesheet you are generating
```
Comment (amyhupe) [inline README.md]: Change:

'If you are including GOV.UK Frontend as part of a stylesheet you are generating'

to:

'If you are including GOV.UK Frontend as part of a stylesheet that you are generating'

File: README.md [hunk 12]
```diff
@@ -175,7 +175,9 @@ Add the CSS and JavaScript code to your HTML template:
 
 ### Usage
 
-Copy and paste code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
+Copy and paste HTML code from the examples in the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/) to use GOV.UK Frontend in your service.
+
+Nunjucks templates are only available when installing GOV.UK Frontend using npm.
```
Comment (amyhupe) [inline README.md]: Change:

'Nunjucks templates are only available when installing GOV.UK Frontend using npm'

to:

'You will only be able to use Nunjucks templates if you install GOV.UK Frontend using npm.'

**Pull request merged**